stage0/gc.go

```diff
@@ -125,9 +164,20 @@ func ForceMountGC(path, uuid string) error {
 		return fmt.Errorf("error getting mounts for pod %s from mountinfo: %v", uuid, err)
 	}
 
+	for i := len(mnts) - 1; i >= 0; i -= 1 {
+		mnt := mnts[i]
+		if strings.Contains(mnt.mode, "shared") || strings.Contains(mnt.mode, "master") {
```

Comment:
True, additional keys could appear in the future. As these are `key:value` pairs adding the colon as in `shared:` and `master:` would solve this issue.